client/app/actions/layout_action_creator.coffee

```diff
@@ -115,18 +124,52 @@ module.exports = LayoutActionCreator =
         # if there is no account, we display the configAccount
         if AccountStore.getAll().size is 0 then 'account.new'
         # else go directly to first account
-        else 'account.mailbox.messages'
+        else 'message.list'
+
+    showMessageList: (data) ->
+        {accountID, mailboxID, messageID, params} = data
+        accountID ?= AccountStore.getSelectedOrDefault()?.get 'id'
+        mailboxID ?= AccountStore.getSelectedMailbox()?.get 'id'
+        unless accountID
+            # TODO : si pas accountID
+            # alors aller à la page de config
+            console.log 'NO ACCOUNT FOUND'
+            return
+
+        # Select Mailbox
+        AppDispatcher.handleViewAction
+            type: ActionTypes.SELECT_ACCOUNT
+            value: {accountID, mailboxID}
 
-    showMessageList: (panelInfo) ->
-        params = panelInfo.parameters
+        # Set message as current
+        if messageID
+            AppDispatcher.handleViewAction
+                type: ActionTypes.MESSAGE_CURRENT
+                value: {messageID}
+
+        # FIXME : normalement,
+        # ça devrait sauvegarder les filtres
+        # Vérifier
+        # FIXME : faire fonctionner
+        # avec 'params' en argument
+        params =
+            pageAfter: '-'
+            sort: '-date'
+            after: '-'
+            before: '-'
+            type: 'nofilter'
+            flag: '-'
+        # Get Messages first
+        unless MessageStore.getCurrentConversation()
+            AppDispatcher.handleViewAction
+                type: ActionTypes.MESSAGE_FETCH_REQUEST
+            AppDispatcher.waitFor[MessageStore.dispatchToken]
 
-        # ensure the proper account is selected
-        {accountID, mailboxID} = params
-        AccountActionCreator.ensureSelected accountID, mailboxID
+        # Display Application
```

Comment:
This will be called every time the user comes back to mesage-list from elsewhere in the app, wont it ?

Why do you want to remove it from initialize.coffee ?